pkg/auth/auth.go

```diff
@@ -114,17 +115,17 @@ func (u *UserAuth) Validate() error {
 	return u.k8sAuth.Validate()
 }
 
-func (u *UserAuth) resolve(groupVersion, kind string) (string, error) {
+func (u *UserAuth) resolve(groupVersion, kind string) (string, bool, error) {
```

Comment:
It might make sense to return a struct in this case with name a namespaced props.